### ISSUE TITLE
Are rotated bounding boxes valid for the current tensorflow object detection api?

### ISSUE THREAD

**Issue by fastlater** [christopher5106/FastAnnotationTool#9]

**Comment by achraf-boussaada** [comment]
Hey !! wondering the same thing !! did you find an answer to your question ?

**Comment by vdalv** [comment]
@fastlater @achraf-boussaada Idk about this tool, but [labelImg](https://github.com/tzutalin/labelImg) supports PascalVOC annotations which the TensorFlow Object Detection API uses. It also supports [YOLO](https://pjreddie.com/darknet/yolo/) annotations.

**Comment by achraf-boussaada** [comment]
@vdalv yes labelImg does the job but you can't label your dataset using rotated bounding boxes and what we're looking for (me at least) is [this](https://github.com/tensorflow/models/issues/3397) and after working with tensorflow I can say that rotating bounding boxes are not supported yet so the issue can be closed.

**Comment by vdalv** [comment]
Ah, yeah I've seen people requesting a rotated bounding box feature for labelImg; I wasn't aware that tensorflow doesn't support them... what about [YOLO/Darknet](https://pjreddie.com/darknet/yolo/)?

**Comment by achraf-boussaada** [comment]
I didn't really try with YOLO but I don't think  it supports the rotated bounding boxes either.
